README.md

```diff
@@ -2,6 +2,32 @@
 
 This project adapts and enhance [Apache James project](https://james.apache.org)
 
+## Useful links
+
+ - We maintain a [CHANGELOG](CHANGELOG.md) and [upgrade instructions](upgrade-instructions.md)
+
+ - [Building + Running the memory server](openpaas-james/apps/memory/README.md)
+
+## Additional features
+
 Additional features includes:
  - JMAP PUSH over AMQP (WIP)
  - JMAP Filters/get and Filters/set (WIP)
+
+## Building the project
+
+This projects uses git submodules to track the latest branch of [the Apache James project](https://james.apache.org)
+
+You need to retrieve Apache sources first:
+
+```
+cd james-project
+git fetch
+git checkout master
```

Comment:
Wrong... see official doc https://git-scm.com/book/en/v2/Git-Tools-Submodules the sections `Cloning a Project with Submodules` and `Working on a Project with Submodules`

When cloning the repo you need to run:
```
git submodule init
git submodule update
```

When you want to update the submodule to its latest state on its original repo:
```
git submodule update --remote
```
State that you need to push in a commit though to keep it up-to-date